@xen-orchestra/web-core/lib/icons/fa-icons.ts

```diff
@@ -135,6 +137,7 @@ export const faIcons = defineIconPack({
   'circle-minus': { icon: faCircleMinus },
   'circle-notch': { icon: faCircleNotch },
   'circle-play': { icon: faCirclePlay },
+  'circle-user': { icon: faCircleUser },
```

Comment:
rename this icon to `account` like in ds/Incons
```suggestion
  'account': { icon: faCircleUser },
```

Reply:
Is it not the name of the FA icon ?

Reply:
In this file, all icons have a faName. However, the circle user icon is placed in the `object-icons.ts` file in the design system. So you can move it in `object-icons.ts`